src/main/resources/templates/generators/overviewFeatures.vm

```diff
@@ -17,10 +17,10 @@
 
 #includeReportInfo()
 
-#includeLead("Features Statistics", "The following graphs show passing and failing statistics for features")
+#if($render_feature_chart && !$all_features.isEmpty())
```

Comment:
Can you redesign it so this if as simple as possible?

Reply:
Move conditional expression check in to features/tags prepareReport(), respectively